src/TfvcMigrator/Program.cs

```diff
@@ -546,6 +548,13 @@ private static void ReportProgress(int changeset, int total, TimedProgress timed
     {
         var builder = ImmutableDictionary.CreateBuilder<string, Identity>(StringComparer.OrdinalIgnoreCase);
 
+        if (!File.Exists(authorsPath))
```

Comment:
Technically there's a race condition between File.Exists here (which could return false) and `File.OpenText` afterwards (which could throw FileNotFoundException), but I won't sweat it.

Reply:
Would you recommend something like to prevent this situation? 
 
```
var lockable = new object();

lock (lockable) { /* chek file existence; create */ }
lock (lockable) { /* read file contents */ }
```

Reply:
The race condition would be a file deletion coming from a separate process. Since the only atomic API .NET provides is to just try to open the file and catch FileNotFoundException, and that's not pleasant to code against, I'm okay saying our tool crashes with this race condition and people can figure out what happened and run it again.